{"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"}]}]}],"kind":"symbol","schemaVersion":{"minor":3,"patch":0,"major":0},"metadata":{"roleHeading":"Case","role":"symbol","modules":[{"name":"Networking"}],"title":"NetworkError.unknown","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"unknown","kind":"identifier"}],"symbolKind":"case","externalID":"s:10Networking12NetworkErrorO7unknownyA2CmF"},"hierarchy":{"paths":[["doc:\/\/Networking\/documentation\/Networking","doc:\/\/Networking\/documentation\/Networking\/NetworkError"]]},"identifier":{"url":"doc:\/\/Networking\/documentation\/Networking\/NetworkError\/unknown","interfaceLanguage":"swift"},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/networking\/networkerror\/unknown"]}],"abstract":[{"type":"text","text":"An indication of an unknown cause."}],"references":{"doc://Networking/documentation/Networking/Response":{"url":"\/documentation\/networking\/response","title":"Response","role":"symbol","fragments":[{"text":"typealias","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"Response"}],"type":"topic","navigatorTitle":[{"kind":"identifier","text":"Response"}],"kind":"symbol","abstract":[{"type":"text","text":"A default "},{"type":"codeVoice","code":"URLSession"},{"type":"text","text":" response tuple."}],"identifier":"doc:\/\/Networking\/documentation\/Networking\/Response"},"doc://Networking/documentation/Networking/NetworkError":{"url":"\/documentation\/networking\/networkerror","title":"NetworkError","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"NetworkError","kind":"identifier"}],"type":"topic","navigatorTitle":[{"text":"NetworkError","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/Networking\/documentation\/Networking\/NetworkError","abstract":[{"type":"text","text":"An error that occurs during "},{"identifier":"doc:\/\/Networking\/documentation\/Networking\/Response","type":"reference","isActive":true},{"type":"text","text":" processing or an underlaying "},{"type":"reference","identifier":"doc:\/\/Networking\/documentation\/Networking","isActive":true},{"text":" error.","type":"text"}]},"doc://Networking/documentation/Networking/NetworkError/unknown":{"url":"\/documentation\/networking\/networkerror\/unknown","title":"NetworkError.unknown","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"}],"type":"topic","kind":"symbol","abstract":[{"text":"An indication of an unknown cause.","type":"text"}],"identifier":"doc:\/\/Networking\/documentation\/Networking\/NetworkError\/unknown"},"doc://Networking/documentation/Networking":{"url":"\/documentation\/networking","role":"collection","identifier":"doc:\/\/Networking\/documentation\/Networking","type":"topic","abstract":[],"title":"Networking","kind":"symbol"}}}